### ISSUE TITLE
Feature request: allow disabling UI parts that are not useful for our workflow

### ISSUE THREAD

**Issue by misha-ridge** [Reviewable/Reviewable#1058]
Please allow disabling some parts of UI that are not useful for our workflow:

- The "merge master" button. We have a bot that takes care of it, following workflow-specific rules.

![Screenshot from 2023-05-12 12-31-43](https://github.com/Reviewable/Reviewable/assets/39945810/6d998019-92e1-4f51-b12d-090ed4dfd09f)

- The green "Merge PR" button whenever a PR can be merged. We have a bot that takes care of it.

It would be nice to also be able to set text in this case. Something like "This PR will be shortly merged automatically".

- LGTM button. We require file reviews instead.

![zz](https://github.com/Reviewable/Reviewable/assets/39945810/12b4f125-7810-4132-af47-ef31c0c350e0)

I have already changed the action for this button, but it does not prevent people from pressing it from time to time, not realizing that it does not do anything.

- The text field next to LGTM button. We have a recurring issue teaching new team members not to use this field: they expect that this field will start a Reviewable discussion that will block the PR, and find it confusing that the PR merges without addressing their comments.

![Screenshot from 2023-05-12 12-32-31](https://github.com/Reviewable/Reviewable/assets/39945810/cd586401-2d1e-4c27-af8c-e6e82a6c5d26)

**Comment by fahhem** [comment]
Hello @misha-ridge ,

Could you explain your workflow a little bit so we can understand how common it is, and if there are simpler or more reusable changes that can be made instead?

It seems the main changes I can deduce are that you don't want us to deal with merging (should the button map to a command to the bot or do all PR's get merged automatically?) and that the top-level discussions are not intuitive to your team.

In regards to disabling merging master, you can set this in your completion condition: https://docs.reviewable.io/repositories.html#disablebranchupdates

Your completion condition can also check for top-level comments (see https://docs.reviewable.io/repositories.html#review-state-input) and warn users via the description (see https://docs.reviewable.io/repositories.html#condition-output) if you'd like to make your team's decision around top-level discussions more discoverable.

What would you like us to do for the merging button?

**Comment by fahhem** [comment]
Thank you for the responses! We'll split this into a few issues and add them to the roadmap. I hope you're liking the product otherwise!

**Comment by misha-ridge** [comment]
Yeah, Reviewable is _the_ reason our team is sticking to GitHub. Would love to see it on GitLab though in future.
